### ISSUE TITLE
Mp3 package does not complie

### ISSUE THREAD

**Issue by lu4p** [faiface/beep#70]
The mp3 Encoder/ Decoder package does currently not compile.

**Comment by FrankBro** [comment]
Builds for me. What's the problem you see?

**Comment by FrankBro** [comment]
I think I managed to reproduce your problem. If you run `go get -u`, I get this error now:

```
# github.com/faiface/beep/mp3
../../go/pkg/mod/github.com/faiface/beep@v1.0.1/mp3/decode.go:98:12: d.d.Close undefined (type *mp3.Decoder has no field or method Close)
```

Not sure why this is happening.

**Comment by faiface** [comment]
This was fixed long time ago, but not included in the v1.0.1 version. I just released v1.0.2 and the fix should be there.
